[2025-01-10T06:16:57.582+0000] {processor.py:157} INFO - Started process (PID=29) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:16:57.586+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:16:57.591+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:16:57.591+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:16:58.102+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:16:58.115+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:16:58.115+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:16:58.129+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:16:58.129+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-09T00:00:00+00:00, run_after=2025-01-10T00:00:00+00:00
[2025-01-10T06:16:58.149+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.571 seconds
[2025-01-10T06:17:28.554+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:28.555+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:17:28.563+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:28.562+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:29.003+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:29.013+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:29.013+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:17:29.023+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:29.023+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-09T00:00:00+00:00, run_after=2025-01-10T00:00:00+00:00
[2025-01-10T06:17:29.029+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.482 seconds
[2025-01-10T06:17:59.165+0000] {processor.py:157} INFO - Started process (PID=33) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:59.166+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:17:59.171+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:59.170+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:59.608+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:17:59.632+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:59.632+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:17:59.641+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:17:59.641+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-09T00:00:00+00:00, run_after=2025-01-10T00:00:00+00:00
[2025-01-10T06:17:59.647+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.490 seconds
[2025-01-10T06:18:30.018+0000] {processor.py:157} INFO - Started process (PID=35) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:18:30.019+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:18:30.026+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:18:30.025+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:18:30.454+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:18:30.464+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:18:30.464+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:18:30.474+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:18:30.474+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:18:30.480+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.467 seconds
[2025-01-10T06:19:00.771+0000] {processor.py:157} INFO - Started process (PID=37) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:00.774+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:19:00.783+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:00.782+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:01.188+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:01.202+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:01.201+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:19:01.213+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:01.213+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:19:01.219+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.454 seconds
[2025-01-10T06:19:31.610+0000] {processor.py:157} INFO - Started process (PID=39) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:31.613+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:19:31.631+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:31.628+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:33.000+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:19:33.026+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:33.025+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:19:33.040+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:19:33.040+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:19:33.048+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 1.449 seconds
[2025-01-10T06:20:03.420+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:03.422+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:20:03.430+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:03.428+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:03.909+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:03.928+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:03.928+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:20:03.937+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:03.937+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:20:03.944+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.536 seconds
[2025-01-10T06:20:34.303+0000] {processor.py:157} INFO - Started process (PID=43) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:34.305+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:20:34.312+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:34.311+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:34.747+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:20:34.758+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:34.757+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:20:34.765+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:20:34.765+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:20:34.771+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.475 seconds
[2025-01-10T06:21:05.011+0000] {processor.py:157} INFO - Started process (PID=45) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:05.012+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:21:05.025+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:05.024+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:05.424+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:05.442+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:05.442+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:21:05.450+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:05.450+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:21:05.457+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.453 seconds
[2025-01-10T06:21:35.743+0000] {processor.py:157} INFO - Started process (PID=47) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:35.744+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:21:35.750+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:35.748+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:36.181+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:36.196+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:36.196+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:21:36.205+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:36.205+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:21:36.212+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.480 seconds
[2025-01-10T06:21:57.034+0000] {processor.py:157} INFO - Started process (PID=49) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:57.035+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:21:57.045+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:57.044+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:57.569+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:21:57.566+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:21:57.569+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:21:57.585+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.557 seconds
[2025-01-10T06:22:27.869+0000] {processor.py:157} INFO - Started process (PID=51) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:22:27.870+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:22:27.874+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:22:27.873+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:22:28.376+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:22:28.375+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:22:28.376+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:22:28.389+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.528 seconds
[2025-01-10T06:24:25.671+0000] {processor.py:157} INFO - Started process (PID=29) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:25.673+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:24:25.687+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:24:25.686+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:26.430+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:24:26.427+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:24:26.431+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:26.448+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.781 seconds
[2025-01-10T06:24:56.872+0000] {processor.py:157} INFO - Started process (PID=31) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:56.875+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:24:56.884+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:24:56.883+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:57.714+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:24:57.712+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:24:57.714+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:24:57.732+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.873 seconds
[2025-01-10T06:25:28.103+0000] {processor.py:157} INFO - Started process (PID=33) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:28.106+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:25:28.111+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:25:28.110+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:28.512+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:25:28.511+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:25:28.513+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:28.531+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.434 seconds
[2025-01-10T06:25:58.869+0000] {processor.py:157} INFO - Started process (PID=35) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:58.870+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:25:58.879+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:25:58.878+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:59.295+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:25:59.294+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/social_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/social_stream_dag.py", line 56, in <module>
    extract_group >> upload_group
    ~~~~~~~~~~~~~~^^~~~~~~~~~~~~~
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 103, in __rshift__
    self.set_downstream(other)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/task_group.py", line 334, in _set_relatives
    task.set_downstream(task_or_task_list)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 261, in set_downstream
    self._set_relatives(task_or_task_list, upstream=False, edge_modifier=edge_modifier)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskmixin.py", line 231, in _set_relatives
    raise AirflowException(
airflow.exceptions.AirflowException: Tried to create relationships between tasks that don't have DAGs yet. Set the DAG for at least one task and try again: [<Task(PythonOperator): extraction_group.extract_reddit_data>, <Task(PythonOperator): upload_group.upload_to_s3>]
[2025-01-10T06:25:59.295+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:25:59.316+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.460 seconds
[2025-01-10T06:26:12.074+0000] {processor.py:157} INFO - Started process (PID=37) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:12.076+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:26:12.080+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:12.079+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:12.934+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:13.236+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.236+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-01-10T06:26:13.247+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.246+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-01-10T06:26:13.250+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.250+0000] {manager.py:499} INFO - Created Permission View: %s
[2025-01-10T06:26:13.253+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.251+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:26:13.268+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.268+0000] {dag.py:2963} INFO - Creating ORM DAG for etl_socialstream_pipeline
[2025-01-10T06:26:13.276+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:13.276+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-09T00:00:00+00:00, run_after=2025-01-10T00:00:00+00:00
[2025-01-10T06:26:13.293+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 1.226 seconds
[2025-01-10T06:26:43.438+0000] {processor.py:157} INFO - Started process (PID=39) to work on /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:43.439+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/social_stream_dag.py for tasks to queue
[2025-01-10T06:26:43.445+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:43.444+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:43.915+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_socialstream_pipeline']) retrieved from /opt/airflow/dags/social_stream_dag.py
[2025-01-10T06:26:43.925+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:43.925+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2025-01-10T06:26:43.934+0000] {logging_mixin.py:151} INFO - [2025-01-10T06:26:43.934+0000] {dag.py:3722} INFO - Setting next_dagrun for etl_socialstream_pipeline to 2025-01-10T00:00:00+00:00, run_after=2025-01-11T00:00:00+00:00
[2025-01-10T06:26:43.941+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/social_stream_dag.py took 0.513 seconds
